runtime/src/cost_model.rs

```diff
@@ -166,18 +171,26 @@ impl CostModel {
             // to keep the same behavior, look for builtin first
             if let Some(builtin_cost) = BUILT_IN_INSTRUCTION_COSTS.get(program_id) {
                 builtin_costs = builtin_costs.saturating_add(*builtin_cost);
-            } else {
-                let instruction_cost = self.find_instruction_cost(program_id);
-                trace!(
-                    "instruction {:?} has cost of {}",
-                    instruction,
-                    instruction_cost
-                );
-                bpf_costs = bpf_costs.saturating_add(instruction_cost);
+            } else if !compute_budget::check_id(program_id) {
+                bpf_costs = bpf_costs.saturating_add(DEFAULT_INSTRUCTION_COMPUTE_UNIT_LIMIT.into());
```

Comment:
Is this effectively the same as compute_budget's "default_units_per_instruction", eg default to be 200K*number_of_ixs.?

Reply:
yeah but only non-builtin, non-compute budget ix